homeassistant/components/flux_led/light.py

```diff
@@ -151,7 +151,6 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         if ipaddr in light_ips:
             continue
         device['name'] = '{} {}'.format(device['id'], ipaddr)
-        device[ATTR_MODE] = MODE_RGBW
```

Comment:
The `device[ATTR_MODE]` value is always read in `__init__()` so this will make it always raise an exception. I think it could work if you set it to `None` instead.

Just a note for others, the mode is set automatically if not explicitly defined: https://github.com/home-assistant/home-assistant/blob/08e93224e0689c9a20a2b6b43057f60348bb7c14/homeassistant/components/flux_led/light.py#L185-L191

(whether that works for all devices, I don't know)